config/plugins-conf.yml

```diff
@@ -65,22 +65,7 @@
 #    key: 111f04e4775bb86c20296530           # required, visit https://www.exchangerate-api.com to get your key, and replace it.
 #    refresh: 3600                           # optional, buffered data within 3600s, recommended for API rate limited data source.
 
-# [optional] Un-comment below lines to select your EVM-like blockchain client websocket service endpoint for crypto plugins.
-# Without configuration, below two plugins connects to default network, an Autonity blockchain RPC node with default
-# contract addresses on the target Autonity blockchain.
-#
-# - name: crypto_airswap                           # optional, it is the plugin file name in the plugin directory.
-#    scheme: "wss"                                 # optional, available values are: "ws" or "wss", default value is "wss".
-#    endpoint: "rpc2.piccadilly.autonity.org"      # optional, default value is a rpc public service of an Autonity network.
-#    ntnTokenAddress: ""                           # optional, NTN ERC20 contract address in hex on the target blockchain
-#    atnTokenAddress: ""                           # optional, Wrapped ATN ERC20 contract address in hex on the target blockchain.
-#    usdcTokenAddress: ""                          # optional, USDC ERC20 contract address in hex on the target blockchain.
-#    swapAddress: ""                               # optional, AirSwap SwapERC20 contract address in hex on the target blockchain.
-
-# - name: crypto_uniswap                           # optional, it is the plugin file name in the plugin directory.
-#    scheme: "wss"                                 # optional, available values are: "http", "https", "ws" or "wss", default value is "wss".
-#    endpoint: "rpc1.piccadilly.autonity.org"      # optional, default value is a rpc public service of an Autonity network.
-#    ntnTokenAddress: ""                           # optional, NTN ERC20 contract address in hex on the target blockchain
-#    atnTokenAddress: ""                           # optional, Wrapped ATN ERC20 contract address in hex on the target blockchain.
-#    usdcTokenAddress: ""                          # optional, USDC ERC20 contract address in hex on the target blockchain.
-#    swapAddress: ""                               # optional, UniSwap factory contract address in hex on the target blockchain.
+# Un-comment below lines to config the RPC endpoint of a Piccadilly Network Full Node for your AMM plugin which sources ATN & NTN market data from an on-chain AMM.
+# - name: crypto_uniswap
+#    scheme: "wss"                                          # optional, available values are: "http", "https", "ws" or "wss", default value is "wss".
+#    endpoint: "rpc-internal-1.piccadilly.autonity.org/ws"  # The default URL might not be stable for public usage, we recommend you to change it!
```

Comment:
endpoint - rpc-internal-1 - this should be the public endpoint rpc1.piccadilly.autonity.org ? @raj-shekhar1 @Jason-Zhangxin-Chen

Reply:
The public RPC endpoints are getting renamed to promote third part RPC providers, discussion [here](https://github.com/autonity/project-execution/issues/379). 
IMO this plugin should use respective validator nodes endpoint like the oracle server, but I am not sure if there is any other impact of this.

Reply:
@cmjc @raj-shekhar1 As discussed with Raj and by according to the discussion list by Raj at https://github.com/autonity/project-execution/issues/379, I updated the default endpoint to `rpc-internal-1.piccadilly.autonity.org./ws` which will be available for default access for the AMM plugin. 
I think we need to let the node operator to decide on which endpoint they want to use, we cannot assume the one used by oracle server can also work for the AMM plugin. Because we don't know what is the network configurations in their setup context. So I would keep it in this way by recommending people to change it, otherwise we use this default config.